### PR TITLE
Declutter some exceptions since we only do py3 + swap pypy for pypy3 in tox

### PR DESCRIPTION
**Pull request by mfalesni**
Do not show the original exceptions that we are "repacking" to shorten the printouts.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{36,37},pypy,codechecks
+envlist = py{36,37},pypy3,codechecks
```
Comment (digitronik) [inline tox.ini]: @mfalesni one more thing If possible to include in same PR l.e. `py35`. As python-3.5 still not retired by python community.

Comment (mfalesni) [reply tox.ini]: Possibly.

Comment (mfalesni) [reply tox.ini]: Actually that is outside of the scope or this PR. This PR just declutters the exceptions and swaps pypy for pypy3 so it only is tested on py3. If they did not update their pypy containers then until recently the latest stable version of pypy3 was the one with compatibility level of Python 3.5.

File: src/widgetastic/browser.py [hunk 2]
```diff
@@ -202,7 +202,7 @@ def _process_locator(locator):
                 if isinstance(loc, WebElement):
                     return loc
             raise LocatorNotImplemented(
-                'You have to implement __locator__ on {!r}'.format(type(locator)))
+                'You have to implement __locator__ on {!r}'.format(type(locator))) from None
```
Comment (mshriver) [inline src/widgetastic/browser.py]: PEP409 was new to me, thanks @mfalesni !!!

**Pull request merged**